dataplane/base.h

```diff
@@ -14,13 +14,48 @@
 namespace dataplane::base
 {
 
+class PortMapper
+{
+	static constexpr tPortId INVALID_PORT_ID = std::numeric_limits<tPortId>::max();
+	uint16_t ports_count_ = 0;
+	tPortId dpdk_ports_[CONFIG_YADECAP_PORTS_SIZE] = {INVALID_PORT_ID}; // logical to dpdk
+	tPortId logical_ports_[std::numeric_limits<tPortId>::max() + 1] = {INVALID_PORT_ID}; // dpdk to logical
```

Comment:
Using `std::array` is moot here. We solely use elements access without bounds checking.

Reply:
But this isn't about boundary checking.
In my opinion, it’s simply better to use `std::array` instead of C-style arrays.
After all, we are writing C++ code, and `std::array` does not compromise on performance.